independent-publisher-connectors/Google BigQuery/apiDefinition.swagger.json

```diff
@@ -0,0 +1,93 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Google BigQuery - Dev",
+    "description": "BigQuery is a completely serverless and cost-effective enterprise data warehouse. It has built-in machine learning and BI that works across clouds, and scales with your data.",
+    "version": "1.0",
+    "contact": {
+      "name": "Ashwani Kumar",
+      "url": "https://about.me/ashwanidv100",
+      "email": "ashwanidv100@gmail.com"
+    }
+  },
+  "host": "www.googleapis.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/bigquery/v2/projects/{projectId}/datasets/{datasetId}": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {
```

Comment:
@ashwanidv100 empty schema is not allowed. Please define a proper schema.

Reply:
I will add this up for successful response.